docs/editors/eclipse.md

```diff
@@ -0,0 +1,27 @@
+---
+id: eclipse
+title: Eclipse
+---
+
+Metals works with Eclipse thanks to the
+[`metals-eclipse`](https://github.com/scalameta/metals-eclipse) plugin.
```

Comment:
Should we add a note that this is a new integration that is under development?

Reply:
Sure, makes sense.